### ISSUE TITLE
Multiple images fail to download with "SSL: WRONG_VERSION_NUMBER"

### ISSUE THREAD

**Issue by antoineco** [lorenzodifuccia/safaribooks#199]
On Ubuntu 19.10 with OpenSSL 1.1.1c, a few images (including the book cover) fail to download with the following message.

```
[#] HTTPSConnectionPool(host='learning.oreilly.com', port=443): Max retries exceeded with url: /library/view/building-event-driven-microservices/9781492057888/assets/Fig-01-xx-BusinessVsTechnological-BCs.png (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:2508)')))
...
[#] Error trying to retrieve this image: Fig-01-xx-BusinessVsTechnological-BCs.png
    From: https://learning.oreilly.com/library/view/building-event-driven-microservices/9781492057888/assets/Fig-01-xx-BusinessVsTechnological-BCs.png
```

Some (most) images download without any issue.

**Comment by TheSnoozer** [comment]
Can you try the workaround described in https://github.com/lorenzodifuccia/safaribooks/issues/163#issuecomment-568148184?

**Comment by antoineco** [comment]
Surprisingly #198 does solve my issue with pictures but not with CSS files. I'll try that workaround, thanks for the pointer!

**Comment by diegorodriguezv** [comment]
Hello. Same problem, same Ubuntu, same OpenSSL.
Changing: `self._start_multiprocessing(self._thread_download_css, self.css)`
For: 
´´´
for css_url in self.css:
    self._thread_download_css(css_url)
´´´
in both ´collect_images()´ and ´collect_css()´ worked, but it seems like downloading the book took considerably more time.

**Comment by diegorodriguezv** [comment]
BTW, this is a duplicate of #163 and #174

**Comment by antoineco** [comment]
I'll close once the fix is merged.

**Comment by lorenzodifuccia** [comment]
Hello @antoineco,
could you try now? I updated the code.
Let me know.

**Comment by antoineco** [comment]
Works like a charm 👍  Thanks @lorenzodifuccia!
